### PR TITLE
Update Close-SmbOpenFile.md

### PR DESCRIPTION
**Pull request by msbemba**
updated syntax, Description for example and added SMB instance

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/winserver2012r2-ps/smbshare/Close-SmbOpenFile.md [hunk 1]
```diff
@@ -53,45 +53,46 @@ Performing operation 'Close-File' on Target '4415226383589'.
...
- 
+
 Confirm 
 Are you sure you want to perform this action? 
 Performing operation 'Close-File' on Target '4415226383569'. 
 [Y] Yes  [A] Yes to All  [N] No  [L] No to All  [S] Suspend  [?] Help (default is "Y"): N
 ```
 
-This example closes one or more files that are open by one of the client identified with the session identifier (ID) 4415226380393 of the SMB server.
+This command closes one or more files that are open by one of the client identified with the session ID 4415226380393 of the SMB server.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/smbshare/Close-SmbOpenFile.md]:
```suggestion
This command closes one or more files that are open by one of the clients identified with the session ID 4415226380393 of the SMB server.
```

Comment (e0i) [reply docset/winserver2012r2-ps/smbshare/Close-SmbOpenFile.md]: cc @msbemba

**Pull request merged**